source/php/App.php

```diff
@@ -31,9 +33,39 @@ public function __construct(private Blade $bladeInstance)
         add_action('restrict_manage_posts', array($this, 'formFilter'));
         add_action('admin_head', array($this, 'jsonSelectedValues'));
 
+        add_action('wp', array($this, 'scheduleRemoveOldCron'));
```

Comment:
Could we run this on some other hook? Running it on `wp` will run it on every request, both frontend and admin.